### PR TITLE
OWConcordance: new output

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Implements #317. 

##### Description of changes
Introduced new output Concordances, which displays the results of Concordance in a Table (which means it can also be saved with Save Data).

##### Includes
- [X] Code changes
- [x] Tests
- [X] Documentation

### CODE REVIEW

File: orangecontrib/text/widgets/owconcordance.py [hunk 1]
```diff
@@ -129,6 +132,7 @@ def data(self, index, role=Qt.DisplayRole):
             const = self.word_index[row][0] in self.colored_rows
             return QColor(236 + 19 * const, 243 + 12 * const, 255)
 
+
```
Comment (robertcv) [inline orangecontrib/text/widgets/owconcordance.py]: Between methods should be only one empty line

File: orangecontrib/text/widgets/owconcordance.py [hunk 2]
```diff
@@ -151,6 +155,21 @@ def matching_docs(self):
...
             return 0
 
+    def get_data(self):
+        domain = Domain([], metas=[StringVariable("Conc. {}".format(
+            self.word)), StringVariable("Document")])
+        data = []
+        docs = []
+        for row in range(self.rowCount()):
+            data.append([])
+            docs.append([])
```
Comment (robertcv) [inline orangecontrib/text/widgets/owconcordance.py]: Not necessary to first append empty table and then override it. Just append the final one.

Comment (ajdapretnar) [reply orangecontrib/text/widgets/owconcordance.py]: Then this data[row].append(str(self.data(index))) won't work. How do I fill the table?

Comment (robertcv) [reply orangecontrib/text/widgets/owconcordance.py]: Create a temporary table to append `str(self.data(index))` and then append `[" ".join(tmp)]` to `data`. For `docs` you can directly append `[self.corpus.titles[self.word_index[row][0]]]` to it.

File: orangecontrib/text/widgets/owconcordance.py [hunk 3]
```diff
@@ -151,6 +155,21 @@ def matching_docs(self):
...
+            for column in range(self.columnCount()):
+                index = self.index(row, column)
+                data[row].append(str(self.data(index)))
+            data[row] = [" ".join(data[row])]
+            docs[row] = [self.corpus.titles[self.word_index[row][0]]]
+        return(Corpus.from_list(domain, np.column_stack((data, docs))))
```
Comment (robertcv) [inline orangecontrib/text/widgets/owconcordance.py]: `from_list` is inherited from `Table` and doesn't correctly initializes `Corpus`. Use `from_table`

**Comment by ajdapretnar** [comment]
@robertcv This is ready for another look.

**Pull request merged**